### DISCUSSION TITLE
One dependency @cypress/request looks like has no support (2 moderate vulnerabilities)

### DISCUSSION THREAD

**Discussion by LubosK** [cypress-io/cypress#27426] [General]
@cypress/request have two vulnerabilities and it looks like theres no one who takes care about it. Is there some way how to solve the issues mentioned below? What should be the approach? Are those issues consider as a part of cypress or its a totally separate project? Thanks in advance for any resposne

1.
More info: https://github.com/advisories/GHSA-72xf-g2v4-qvf3
Patch available
Opened issue: https://github.com/cypress-io/request/issues/31
Opened PR: https://github.com/cypress-io/request/pull/32
2.
More info: https://github.com/advisories/GHSA-p8p7-x288-28g6
No patch available 
Opened issue: https://github.com/cypress-io/request/issues/27
PR: https://github.com/cypress-io/request/pull/30

**Comment by jordanpowell88** [comment]
We released version `2.88.12` today which patches this security vulnerability https://www.npmjs.com/package/@cypress/request/v/2.88.12

**Reply by LubosK** [reply]
Thank you, @jordanpowell88 !

**Comment by LubosK** [comment]
Hi @jordanpowell88 , I just checked new version, but it has solved only `tough-cookie` issue. The second one, is still the issue https://github.com/cypress-io/request/issues/27
![image](https://github.com/cypress-io/cypress/assets/15995793/821f3c20-870c-4151-bc08-a5259525f99b)
